tests/vcd_system.py

```diff
@@ -0,0 +1,21 @@
+import random
+import string
+import unittest
+
+from pyvcloud.vcd.system import System
+from pyvcloud.vcd.test import TestCase
+
+
+class TestSystem(TestCase):
+    def test_create_org(self):
+        sys_admin = self.client.get_admin()
+        system = System(self.client, admin_resource=sys_admin)
+        org_name = "orgName".join(
```

Comment:
move these static strings to the config file.

Reply:
I didn't want org_name getting confused with org in config.yml